src/move.sol

```diff
@@ -62,7 +62,10 @@ contract DaiMove {
     function hope(address guy) public { can[msg.sender][guy] = 1; }
     function nope(address guy) public { can[msg.sender][guy] = 0; }
     function move(address src, address dst, uint wad) public {
-        require(src == msg.sender || can[src][msg.sender] == 1);
-        vat.move(bytes32(bytes20(src)), bytes32(bytes20(dst)), mul(ONE, wad));
+        move(bytes32(bytes20(src)), bytes32(bytes20(dst)), wad);
```

Comment:
Does it make sense to do the `bytes20` here? It will be executed in the called function anyway.

Reply:
You need to parse to `bytes20` or to `uint256` before parsing to `bytes32`. This is a requirement for solc 0.5. If you parse with `bytes` will align the address to the left, otherwise to the right.